diff --git a/node_modules/@zamiell/typedoc-plugin-not-exported/dist/main.js b/node_modules/@zamiell/typedoc-plugin-not-exported/dist/main.js
index 583112f..c1d9e9f 100644
--- a/node_modules/@zamiell/typedoc-plugin-not-exported/dist/main.js
+++ b/node_modules/@zamiell/typedoc-plugin-not-exported/dist/main.js
@@ -6,18 +6,18 @@
  * And: https://github.com/tomchen/typedoc-plugin-not-exported
  * CC0
  */
-Object.defineProperty(exports, "__esModule", { value: true });
-const typedoc_1 = require("typedoc"); // version 0.20.16+
+Object.defineProperty(exports, "__esModule", {value: true});
+const typedoc_1   = require("typedoc"); // version 0.20.16+
 const ModuleFlags = typedoc_1.TypeScript.SymbolFlags.ValueModule | typedoc_1.TypeScript.SymbolFlags.NamespaceModule;
-exports.load = function (application) {
+exports.load      = function (application) {
     /** @type {Map<Reflection, Set<TypeScript.SourceFile>>} */
     const checkedForModuleExports = new Map();
-    let includeTag = 'notExported';
+    let includeTag                = 'notExported';
     application.options.addDeclaration({
-        name: 'includeTag',
-        help: '[typedoc-plugin-not-exported] Specify the tag name for non-exported member to be imported under',
-        defaultValue: includeTag,
-    });
+                                           name:         'includeTag',
+                                           help:         '[typedoc-plugin-not-exported] Specify the tag name for non-exported member to be imported under',
+                                           defaultValue: includeTag,
+                                       });
     application.converter.on(typedoc_1.Converter.EVENT_BEGIN, () => {
         const includeTagTemp = application.options.getValue('includeTag');
         if (typeof includeTagTemp === 'string') {
@@ -28,6 +28,7 @@ exports.load = function (application) {
     application.converter.on(typedoc_1.Converter.EVENT_END, () => {
         checkedForModuleExports.clear();
     });
+
     function lookForFakeExports(context, reflection) {
         // Figure out where "not exports" will be placed, go up the tree until we get to
         // the module where it belongs.
@@ -35,17 +36,22 @@ exports.load = function (application) {
         while (!targetModule.kindOf(typedoc_1.ReflectionKind.Module | typedoc_1.ReflectionKind.Project)) {
             targetModule = targetModule.parent;
         }
-        const moduleContext = context.withScope(targetModule);
-        const reflSymbol = context.project.getSymbolFromReflection(reflection);
-        if (!reflSymbol) {
-            // Global file, no point in doing anything here. TypeDoc will already
-            // include everything declared in this file.
-            return;
-        }
-        for (const declaration of reflSymbol.declarations || []) {
-            checkFakeExportsOfFile(declaration.getSourceFile(), moduleContext);
+        try {
+            const moduleContext = context.withScope(targetModule);
+            const reflSymbol    = context.getSymbolFromReflection(reflection);
+            if (!reflSymbol) {
+                // Global file, no point in doing anything here. TypeDoc will already
+                // include everything declared in this file.
+                return;
+            }
+            for (const declaration of reflSymbol.declarations || []) {
+                checkFakeExportsOfFile(declaration.getSourceFile(), moduleContext);
+            }
+        } catch (e) {
+
         }
     }
+
     function checkFakeExportsOfFile(file, context) {
         const moduleSymbol = context.checker.getSymbolAtLocation(file);
         // Make sure we are allowed to call getExportsOfModule
@@ -58,13 +64,13 @@ exports.load = function (application) {
             return;
         checkedScopes.add(file);
         const exportedSymbols = context.checker.getExportsOfModule(moduleSymbol);
-        const symbols = context.checker
-            .getSymbolsInScope(file, typedoc_1.TypeScript.SymbolFlags.ModuleMember)
-            .filter((symbol) => {
-            var _a;
-            return ((_a = symbol.declarations) === null || _a === void 0 ? void 0 : _a.some((d) => d.getSourceFile() === file)) &&
-                !exportedSymbols.includes(symbol);
-        });
+        const symbols         = context.checker
+                                       .getSymbolsInScope(file, typedoc_1.TypeScript.SymbolFlags.ModuleMember)
+                                       .filter((symbol) => {
+                                           var _a;
+                                           return ((_a = symbol.declarations) === null || _a === void 0 ? void 0 : _a.some((d) => d.getSourceFile() === file)) &&
+                                                  !exportedSymbols.includes(symbol);
+                                       });
         for (const symbol of symbols) {
             if (symbol
                 .getJsDocTags()
@@ -73,6 +79,7 @@ exports.load = function (application) {
             }
         }
     }
+
     // Fix for the new TypeDoc JSDoc tag linting.
     application.on(typedoc_1.Application.EVENT_BOOTSTRAP_END, () => {
         const modifiers = application.options.getValue('modifierTags');
